graphql/parse.lua

```diff
@@ -1,4 +1,4 @@
-local lpeg = require 'lpeg'
+local lpeg = require 'lulpeg'
```

Comment:
Please, see #3 on this.

Reply:
Discussed online and decided to stick to lulpeg for now.
Updated PR description.